themes/all.ini

```diff
@@ -2,7 +2,7 @@ catppuccin/latte/name = Catppuccin Latte
 catppuccin/latte/background = #eff1f5
 catppuccin/latte/currentline = #ccd0da
 catppuccin/latte/currentcell = #eff1f5
-catppuccin/latte/occurrence = #b0c7f5
```

Comment:
All changes to `occurence` as seen below are due to the colors for it not being valid Catppuccin colors. I used the closest color to it though, which is `surface1`.